router/router.go

```diff
@@ -430,24 +420,24 @@ const (
 	OpenControlChannel  byte = 4
 )
 
-func (router *Router) RegisterDefaultDebugOps() {
-	router.debugOperations[DumpForwarderTables] = router.debugOpWriteForwarderTables
-	router.debugOperations[UpdateRoute] = router.debugOpUpdateRouter
-	router.debugOperations[CloseControlChannel] = router.debugOpCloseControlChannel
-	router.debugOperations[OpenControlChannel] = router.debugOpOpenControlChannel
+func (self *Router) RegisterDefaultDebugOps() {
+	self.debugOperations[DumpForwarderTables] = self.debugOpWriteForwarderTables
+	self.debugOperations[UpdateRoute] = self.debugOpUpdateRouter
+	self.debugOperations[CloseControlChannel] = self.debugOpCloseControlChannel
+	self.debugOperations[OpenControlChannel] = self.debugOpOpenControlChannel
 }
 
-func (router *Router) RegisterDebugOp(opId byte, f func(c *bufio.ReadWriter) error) {
-	router.debugOperations[opId] = f
+func (self *Router) RegisterDebugOp(opId byte, f func(c *bufio.ReadWriter) error) {
```

Comment:
I don't mind `self`. Got used to it while working in C#. Goes against the language creator's style guide so I'm always torn on it.  No change needed, just musing. 

Did you disable the default warning that GoLand shows?

Reply:
yes. @michaelquigley started using it and I got on board because it saves time trying to think of what to call everything. I read their rationale for why it was wrong, and thought it was pretty weak :)